### DISCUSSION TITLE
Need help Please

### DISCUSSION THREAD

**Discussion by Opus44o** [kalkmann/Legion-5600H-Hackintosh#28] [Q&A]
Good morning 
first of all thank you for your work 
I installed Sonoma on my Lenovo (15ACH6A) 
I copied your EFI folder into my EFI partition but nothing works 
no wifi, AMD video card not recognized 

Can you help me please?

**Comment by kalkmann** [comment]
> Good morning first of all thank you for your work I installed Sonoma on my Lenovo (15ACH6A) I copied your EFI folder into my EFI partition but nothing works no wifi, AMD video card not recognized
> 
> Can you help me please?

Same Legion model? Strange. How do you know the wifi doesn't work if the dGPU didn't work? Do you have the "only discrete graphics" option enabled in Bios? What is your Bios version?
I don't use the original wifi adapter as described in the readme.

**Reply by Opus44o** [reply]
thank you for your reply 
here is a photo of the BIOS 
I updated the BIOS 
![IMG_4762](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/170352326/6fc603e8-b196-4d51-b037-38dad4f74924)
I have an image but with 8MB of video memory

**Reply by Opus44o** [reply]
![IMG_4763](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/170352326/6aa5f32f-9bfc-4d50-a752-6ebdc1a31546)

**Reply by kalkmann** [reply]
It's not the same model. Mine has an AMD RX6600m dGPU, yours has NVIDIA, which is not supported by MacOS. You will have to follow another path, using the Nootedred kext to enable the integrated video.
But I'm sorry I don't have experience in this type of setup.

Yesterday I answered a question similar to your setup, with a link to a similar setup (Lenovo Yoga with nvidia dgpu).

Good luck!

**Reply by kalkmann** [reply]
Check this reference: https://github.com/ducviet5138/Hackintosh-Legion-5
